README.md

```diff
@@ -17,15 +17,16 @@
 <img src="resources/pack-build.gif" width="600px" />
 
 ## Getting Started
-
```

Comment:
FWIW, new lines after headers are a matter of preference. I can see myself adding it back in another PR. :D

Reply:
Haha that's fair. I guess we just need a style guide 😅